src/main/java/com/purbon/kafka/topology/clusterstate/YAMLStateProcessor.java

```diff
@@ -0,0 +1,54 @@
+package com.purbon.kafka.topology.clusterstate;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLFactory;
+import java.io.Reader;
+import java.io.Writer;
+
+public class YAMLStateProcessor implements StateProcessor {
```

Comment:
quick question: Why YAML and not JSON ? curiosity 😄

Reply:
Just not to introduce a new format. yaml is used for configuration already.